
<!DOCTYPE html>
<!--[if IEMobile 7 ]><html class="no-js iem7"><![endif]-->
<!--[if lt IE 9]><html class="no-js lte-ie8"><![endif]-->
<!--[if (gt IE 8)|(gt IEMobile 7)|!(IEMobile)|!(IE)]><!--><html class="no-js" lang="en"><!--<![endif]-->
<head>
  <meta charset="utf-8">
  <title>Bing & KK</title>
  <meta name="author" content="BingW">

  
  <meta name="description" content="Annotaion Structural Annotation - undaries of features of interest (coding regions, regulatory elements, functional RNAs, etc). Ab initio - &hellip;">
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://BingW.github.com/">
  <link href="/favicon.png" rel="icon">
  <link href="/stylesheets/screen.css" media="screen, projection" rel="stylesheet" type="text/css">
  <script src="/javascripts/modernizr-2.0.js"></script>
  <script src="/javascripts/ender.js"></script>
  <script src="/javascripts/octopress.js" type="text/javascript"></script>
  <link href="/atom.xml" rel="alternate" title="Bing & KK" type="application/atom+xml">
  <!--Fonts from Google"s Web font directory at http://google.com/webfonts -->
<link href="http://fonts.googleapis.com/css?family=PT+Serif:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">

<!-- mathjax config similar to math.stackexchange -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
      }
    });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Queue(function() {
        var all = MathJax.Hub.getAllJax(), i;
        for(i=0; i < all.length; i += 1) {
            all[i].SourceElement().parentNode.className += ' has-jax';
        }
    });
</script>

<script type="text/javascript"
   src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>

  

</head>

<body   >
  <header role="banner"><hgroup>
  <h1><a href="/">Bing & KK</a></h1>
  
    <h2>Conding with Zen</h2>
  
</hgroup>

</header>
  <nav role="navigation"><ul class="subscription" data-subscription="rss">
  <li><a href="/atom.xml" rel="subscribe-rss" title="subscribe via RSS">RSS</a></li>
  
</ul>
  
<form action="http://google.com/search" method="get">
  <fieldset role="search">
    <input type="hidden" name="q" value="site:BingW.github.com" />
    <input class="search" type="text" name="q" results="0" placeholder="Search"/>
  </fieldset>
</form>
  
<ul class="main-navigation">
  <li><a href="/">Blog</a></li>
  <li><a href="/blog/archives">Archives</a></li>
</ul>

</nav>
  <div id="main">
    <div id="content">
      <div class="blog-index">
  
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/24/structural-functional-annotation/">Structural Functional Annotation</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-24T14:27:00-05:00" pubdate data-updated="true">Sep 24<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><h4 id="annotaion">Annotaion</h4>

<ul>
  <li>Structural Annotation - undaries of features of 
interest (coding regions, regulatory elements, functional 
RNAs, etc). 
    <ul>
      <li>Ab initio - computationally predicted</li>
      <li>Comparative - based on similarity to other genes or genomes</li>
      <li>Experimental - transcript sequencing</li>
    </ul>
  </li>
  <li>Functional Annotation - attaching meaning to the features 
(names, product, activity, biological role, etc.)
    <ul>
      <li>Sequence homology</li>
      <li>Structural similarity or structural features</li>
      <li>Experimental data - gene or protein expression patterns</li>
    </ul>
  </li>
</ul>

<h5 id="errors">Errors</h5>

<ul>
  <li>Structural Annotation Errors
    <ul>
      <li>Wrong coordinates for the gene (exons, regulatory sites, 
etc.)</li>
      <li>False positives - not really a gene (etc.) at all</li>
      <li>False negatives - true genes (etc.) missing</li>
    </ul>
  </li>
  <li>Functional Annotation Errors
    <ul>
      <li>Overly specific function attributed</li>
      <li>Wrong function attributed</li>
      <li>True (known) function missed</li>
      <li>Typos and misspellings</li>
    </ul>
  </li>
</ul>

<h5 id="quality">Quality</h5>

<ul>
  <li>
    <p>Model organisms have content rich structural and 
functional annotations developed and updated over 
time combining <code>computational predictions</code> with 
<code>experimental data</code> and <code>manual curation</code>.</p>
  </li>
  <li>
    <p>Non-model organisms can be well annotated, but 
many are not.</p>
  </li>
  <li>
    <p>Standards for quality and content are still under 
active development and adherence to the standards 
is voluntary.</p>
  </li>
</ul>

<h4 id="annotation-source">annotation source</h4>

<ul>
  <li>From an INSD (International Nucleotide 
Sequence Database) provider
    <ul>
      <li>Genbank</li>
      <li>ENA</li>
      <li>DDBJ</li>
    </ul>
  </li>
  <li>From a “Third Party” provider
    <ul>
      <li>GMOD</li>
      <li>Other specialized databases</li>
    </ul>
  </li>
</ul>

<h4 id="approaches">approaches</h4>

<ul>
  <li>Ab initio
    <ul>
      <li>Based exclusively on computational models</li>
      <li>Error prone, esp. for eukaryotes</li>
      <li>Generally requires manual clean up</li>
    </ul>
  </li>
  <li>Comparative
    <ul>
      <li>Find genes corresponding to sequenced transcripts</li>
      <li>Find the genes already predicted for a closely related 
 organism</li>
    </ul>
  </li>
</ul>

<h4 id="hmms-and-annotation">HMMs and annotation</h4>

<ul>
  <li>Markov models are used in many bioinformatics 
applications</li>
  <li>In particular, Hidden Markov Models are used for
    <ul>
      <li>Ab initio gene prediction</li>
      <li>Ensemble methods for combining results of multiple gene 
 prediction methods</li>
      <li>Protein family similarity searches</li>
      <li>Prediction of other types of features (such as regulatory 
 binding sites)</li>
    </ul>
  </li>
</ul>

<h5 id="using-hmms">Using HMMs</h5>

<ul>
  <li>Given the parameters of the model, compute the probability 
    of a particular output sequence. This problem is solved by the 
    <em>forward algorithm</em>.</li>
  <li>Given the parameters of the model, find the most likely 
    sequence of hidden states that could have generated a given 
    output sequence. This problem is solved by the <em>Viterbi 
    algorithm</em>.</li>
  <li>Given an output sequence or a set of such sequences, find the 
    most likely set of state transition and output probabilities. In 
    other words, train the parameters of the HMM given a 
    dataset of sequences. This problem is solved by the <em>BaumWelch algorithm</em>.</li>
</ul>

<p><a href="http://www.nature.com/nbt/journal/v22/n10/full/nbt1004-1315.html">HMM introduction</a></p>

<h5 id="models-and-training-sets">Models and Training Sets</h5>

<ul>
  <li>The parameters of models are determined from a 
    training set of known positives and negatives, for 
    example a set of known genes, and regions known to 
    not encode genes.</li>
  <li>Size and quality of training sets impacts the 
    performance of models</li>
  <li>Size and quality of training sets is related to the 
    specific genome under investigation</li>
</ul>

<h4 id="the-gene-ontology-go">The Gene Ontology (GO)</h4>

<ul>
  <li>Goal = consistent annotation of gene products within 
    and between organisms</li>
  <li>Gene Ontology Consortium began as a collaboration 
    among model organism dbs (FlyBase, SGD, MGD). 
    Now includes larger number of members and 
    interest groups</li>
  <li>Ontology = A formal representation of concepts and 
    the relationships among them</li>
</ul>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/20/biochem-note-1/">Biochem Note 1</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-20T17:07:00-05:00" pubdate data-updated="true">Sep 20<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content">
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/19/genome-alignment-and-rna-seq/">Genome Alignment and RNA-seq</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-19T14:26:00-05:00" pubdate data-updated="true">Sep 19<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><h4 id="genome-alignment">Genome Alignment</h4>
<ul>
  <li>Pair-wise
    <ul>
      <li>Align two genomes</li>
      <li>Example: MUMmer</li>
    </ul>
  </li>
  <li>Multiple or complex samples and a reference genome
    <ul>
      <li>All of one genome plus whatever parts match from the other  genome(s)</li>
      <li>Example: PIPs</li>
    </ul>
  </li>
  <li>Multiple alignment
    <ul>
      <li>All of all the genomes</li>
      <li>Example:  Mauve</li>
    </ul>
  </li>
</ul>

<h5 id="mummerhttpmummersourceforgenet"><a href="http://mummer.sourceforge.net/">MUMmer</a></h5>

<p>Fast pair-wise comparison of draft or complete 
genomes using nucleotide or 6-frame translated 
sequences</p>

<p>? Suffix Tree</p>

<h5 id="percent-identity-plotpip">Percent Identity Plot(PIP)</h5>

<p>quality draft</p>

<h5 id="mauve-multiple-genome-aligner">Mauve multiple Genome Aligner</h5>

<p>Progressive Mauve alignment</p>

<h4 id="rna-seq-goals-and-computational-challenges">RNA-seq Goals and Computational Challenges</h4>

<p>Goal:
+ Annotation
+ Quantification of genes
+ Quantification of isoforms</p>

<p>Challenges:
+ Read mapping
+ Transcriptome reconstruction
+ Expression quantification</p>

<h5 id="domain-specific-challenges">Domain-specific challenges</h5>

<ul>
  <li>Eukaryotes
    <ul>
      <li>Processed and unprocessed RNAs in the sample</li>
      <li>Quality of genome sequences</li>
      <li>Quality of gene (and alternate splicing) annotations</li>
      <li>Isoforms</li>
    </ul>
  </li>
  <li>Prokaryotes
    <ul>
      <li>rRNA</li>
      <li>Gene density</li>
      <li>Operons</li>
    </ul>
  </li>
</ul>

<h5 id="popular-in-wisc">popular in WISC</h5>

<ul>
  <li>Bowtie, Tophat, Cufflinks, CuffDiff</li>
  <li>RSEM, EBSeq</li>
</ul>

<h5 id="read-mapping">Read mapping</h5>

<ul>
  <li>Unspliced aligners: 
    <ul>
      <li>Seed based (Maq, Stampy)</li>
      <li>Burrows Wheeler based (BWA, Bowtie, etc.)</li>
      <li>How similar is your reference genome?</li>
      <li>Best for mapping against cDNA databases from well characterized reference sequences</li>
      <li>Can only recognize known exons and isoforms</li>
    </ul>
  </li>
  <li>Spliced aligners:
    <ul>
      <li>Exon first (MapSplice, SpliceMap, TopHat)
        <ul>
          <li>unspliced alignment, split reads and repeat</li>
          <li>Most efficient if relatively few reads span junctions</li>
          <li>Can miss splice sites if reads map elsewhere in their entirety
 – Seed and Extend (GSNAP, QPALMA)</li>
          <li>Split all reads into seeds, map, merge and extend</li>
          <li>Slower, but detect more splice junctions</li>
          <li>Superior with polymorphic species</li>
        </ul>
      </li>
    </ul>
  </li>
</ul>

<h5 id="transcriptome-reconstruction-challenges">Transcriptome Reconstruction Challenges</h5>

<ul>
  <li>Genes vary in expression level, some may be represented by very few reads</li>
  <li>Reads can come from either mature mRNA or a precursor</li>
  <li>When there are multiple isoforms, how can you tell which gave rise to a particular read?  </li>
</ul>

<h5 id="genome-guided-vs-genome-independent-reconstruction">Genome-guided vs. Genome Independent Reconstruction</h5>

<ul>
  <li>Genome-guided reconstruction
    <ul>
      <li>Exon identification (G.mor.se)</li>
      <li>Genome-guided assembly graph (CuffLinks, Scripture)</li>
    </ul>
  </li>
  <li>Performance varies with the expression level</li>
</ul>

<h5 id="qunantifcation-overview">Qunantifcation overview</h5>

<ul>
  <li><strong>TODO</strong> fig</li>
</ul>

<h5 id="normalization">Normalization</h5>

<ul>
  <li>Genes vary in length - with random fragmentation, long genes are expected to have more reads than short genes expressed at the same level</li>
  <li>Sequence “runs”/ experiments differ in total number of reads collected</li>
  <li>RPKM = Reads per kilobase of transcript per million mapped reads (FPKM = fragments…)</li>
  <li>Many other more sophisticated normalization methods</li>
</ul>

<p>`</p>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/18/colorful-python/">让你的python亮起来</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-18T12:12:00-05:00" pubdate data-updated="true">Sep 18<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p>昨晚写了个程序给老婆看，老婆表示很好，就是在terminal里面只有白色单调了点，所以最好要有颜色。然后求助万能的google未果，只好求助更加万能的<a href="http://stackoverflow.com/questions/287871/print-in-terminal-with-colors-using-python">stakeoverflow</a>，果然，顶得人不少，有157票赞同这个方案：</p>

<div class="bogus-wrapper"><notextile><figure class="code"> <div class="highlight"><table><tr><td class="gutter"><pre class="line-numbers"><span class="line-number">1</span>
<span class="line-number">2</span>
<span class="line-number">3</span>
<span class="line-number">4</span>
<span class="line-number">5</span>
<span class="line-number">6</span>
<span class="line-number">7</span>
<span class="line-number">8</span>
<span class="line-number">9</span>
<span class="line-number">10</span>
<span class="line-number">11</span>
<span class="line-number">12</span>
<span class="line-number">13</span>
<span class="line-number">14</span>
</pre></td><td class="code"><pre><code class="python"><span class="line">    <span class="k">class</span> <span class="nc">bcolors</span><span class="p">:</span>
</span><span class="line">        <span class="n">HEADER</span> <span class="o">=</span> <span class="s">&#39;</span><span class="se">\033</span><span class="s">[95m&#39;</span> <span class="c">#pink</span>
</span><span class="line">        <span class="n">OKBLUE</span> <span class="o">=</span> <span class="s">&#39;</span><span class="se">\033</span><span class="s">[94m&#39;</span> <span class="c">#blue</span>
</span><span class="line">        <span class="n">OKGREEN</span> <span class="o">=</span> <span class="s">&#39;</span><span class="se">\033</span><span class="s">[92m&#39;</span> <span class="c">#green</span>
</span><span class="line">        <span class="n">WARNING</span> <span class="o">=</span> <span class="s">&#39;</span><span class="se">\033</span><span class="s">[93m&#39;</span> <span class="c">#yellow</span>
</span><span class="line">        <span class="n">FAIL</span> <span class="o">=</span> <span class="s">&#39;</span><span class="se">\033</span><span class="s">[91m&#39;</span> <span class="c">#red</span>
</span><span class="line">        <span class="n">ENDC</span> <span class="o">=</span> <span class="s">&#39;</span><span class="se">\033</span><span class="s">[0m&#39;</span> <span class="c">#white</span>
</span><span class="line">
</span><span class="line">    <span class="k">print</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">HEADER</span> <span class="o">+</span> <span class="s">&quot;Header&quot;</span> <span class="o">+</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">ENDC</span>
</span><span class="line">    <span class="k">print</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">OKBLUE</span> <span class="o">+</span> <span class="s">&quot;OK&quot;</span> <span class="o">+</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">ENDC</span>
</span><span class="line">    <span class="k">print</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">OKGREEN</span> <span class="o">+</span> <span class="s">&quot;another OK&quot;</span> <span class="o">+</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">ENDC</span>
</span><span class="line">    <span class="k">print</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">WARNING</span> <span class="o">+</span> <span class="s">&quot;Warning&quot;</span> <span class="o">+</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">ENDC</span>
</span><span class="line">    <span class="k">print</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">FAIL</span> <span class="o">+</span> <span class="s">&quot;Fail&quot;</span> <span class="o">+</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">ENDC</span>
</span><span class="line">    <span class="k">print</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">ENDC</span> <span class="o">+</span> <span class="s">&quot;Endc&quot;</span> <span class="o">+</span> <span class="n">bcolors</span><span class="o">.</span><span class="n">ENDC</span>
</span></code></pre></td></tr></table></div></figure></notextile></div>

<p>运行结果：</p>

<p><img src="http://bingw.github.com/images/color_python.png" alt="colorfor_python" /></p>

<p>注意事项：</p>

<ul>
  <li>后面的bcolors.ENDC最好加上，不然你的terminal就成那个颜色了，用ENDC是因为个人的terminal字体本来就是白色的……</li>
  <li>貌似只在macOS，linux系统上可行，windows上需要开启<a href="http://support.microsoft.com/kb/101875">ansi.sys</a></li>
  <li>如果需要更复杂的特性，可以阅读python document的<a href="http://docs.python.org/howto/curses.html">curses</a></li>
</ul>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/17/variant-discovery-using-ngs/">Variant Discovery Using NGS</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-17T14:33:00-05:00" pubdate data-updated="true">Sep 17<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><h4 id="types-of-variants">Types of Variants</h4>

<ul>
  <li>Simple Single Nucleotide Polymorphisms (SNPs)</li>
  <li>Multiple adjacent or clustered polymorphisms</li>
  <li>Insertions and deletions (Indels)</li>
  <li>Inversions</li>
  <li>Rearrangements</li>
  <li>Copy number variation</li>
</ul>

<h4 id="separating-true-variants-from-artifacts">Separating true variants from artifacts</h4>

<ul>
  <li>Ignore suspicious regions – areas where too 
many reads mapped given the expected 
coverage</li>
  <li>Require three reads on each strand</li>
</ul>

<h4 id="tradeoffs-of-sp-vs-sn">Tradeoffs of Sp VS Sn</h4>

<ul>
  <li>Sn = TP/(TP + FN)</li>
  <li>Sn = Sensitivity: what portion of the total number 
     of real variants did you detect?</li>
  <li>Sp = TN/(TN + FP)</li>
  <li>Sp = Specificity: what portion of the variants you 
     discovered are really variants, not false 
     positives?</li>
</ul>

<h4 id="framework-for-variation">Framework for variation</h4>

<ol>
  <li>NGS data processing </li>
  <li>variant discovery and genotyping</li>
  <li>integrative analysis
see <a href="http://bingw.github.com/blog/2012/09/17/depristo-variationdiscovery/">here</a></li>
</ol>

<h4 id="sambam-and-samtools">SAM/BAM and SAMtools</h4>

<ul>
  <li>see <a href="http://en.wikipedia.org/wiki/SAMtools">SAMtools</a></li>
  <li>see <a href="http://genome.sph.umich.edu/wiki/SAM">SAM</a></li>
</ul>

<h4 id="final-objective--vcf">Final objective = VCF</h4>

<p><a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">format</a> </p>

<p>Q1: How to determine a real SNP?<br />
Q2: we canot assure the sencetivity and specfity are same in Novel and kwon data?</p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/17/depristo-variationdiscovery/">DePristo_VariationDiscovery</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-17T00:26:00-05:00" pubdate data-updated="true">Sep 17<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><h3 id="a-frame-work-for-variation-discovery-and-genotyping-using-next-genetation-dna-sequencing-data">A frame work for variation discovery and genotyping using next-genetation DNA sequencing data</h3>
<p><em>Mark A DePristo .etc</em></p>

<h4 id="abstract">Abstract</h4>
<p>A frame work to discovery and genotype variation among multiple samples <br />
across five sequencing technologies and three canonical experimental design.</p>

<p>process:</p>

<ul>
  <li>initial read mapping</li>
  <li>local alignment around indels</li>
  <li>base quality score recalibration</li>
  <li>SNP discovery and genotyping to find all potential variants</li>
  <li>machine learing to separate true segreating variation from machnie artifacts
common to next-generation sequencing tech.</li>
</ul>

<h4 id="introduction">Introduction</h4>
<p>three datasets:
<img src="http://bingw.github.com/images/DePristo_table1.png" alt="NGS datasets analyzed" /></p>

<h4 id="results">Results</h4>
<p><img src="http://bingw.github.com/images/DePristo_fig1.png" alt="Three-part conceptual framework" /></p>

<p>instantiated in Genome Analysis Toolkit <a href="http://genome.cshlp.org/content/20/9/1297"><abbr title="Genome Analysis Toolkit">GATK</abbr> ref</a>
<a href="http://www.broadinstitute.org/gatk/">Toolkit</a></p>

<h4 id="applying-the-analysis-pipeline-to-hiseq28-mb-exome-capturelow-pass-4x-sequencing">Applying the analysis pipeline to Hiseq/28-Mb exome capture/low-pass (4x) sequencing</h4>
<p><img src="http://bingw.github.com/images/DePristo_table2.png" alt="Raw to racalibrated" /></p>

<p><img src="http://bingw.github.com/images/DePristo_fig4.png" alt="Results of variant quality recalibration" /></p>

<p>Some concepts:<abbr title="transition(A<->G,T<->C) / transversion(others..)&#8221;>Ti/Tv</abbr>,<abbr title="linkage disequilibrium">LD</abbr>,<abbr title="copy number variations">CNVs</abbr>,<abbr title="next-generation sequencing">NGS</abbr>,<abbr title="1000 Genomes Project">1KG</abbr>,<abbr title="Genome Analysis Toolkit">GATK</abbr>,<abbr title="false discovery rate">FDR</abbr>,<abbr title="Single Nucleotide Polymorphism Database">dbSNP</abbr>,<abbr title="whole genome shotgun">WGS</abbr>,<abbr title="non-reference discrepancy">NRD</abbr>,<abbr title="non-reference sensitivity">NR</abbr>,<abbr title="multiple sequence realignment">MSA</abbr>,<abbr title="Root-mean-square errors">RMSE</abbr>,<abbr title="HapMap3">HM3</abbr></p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/13/yaml-format/">YAML Format</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-13T16:39:00-05:00" pubdate data-updated="true">Sep 13<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p>start with <code>---</code> end with <code>---</code> at the head of your document:</p>

<ul>
  <li>layout: post</li>
  <li>title: “Hello World”</li>
  <li>date: 2012-06-19 05:20 </li>
  <li>comments: true </li>
  <li>categories: test</li>
  <li>author: Bing Wang</li>
  <li>
    <p>published: true</p>
  </li>
  <li>external_url: </li>
  <li>sharing: true</li>
  <li>footer: true</li>
</ul>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/13/kramdown-syntax/">Kramdown Syntax</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-13T14:42:00-05:00" pubdate data-updated="true">Sep 13<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><a href="http://kramdown.rubyforge.org/quickref.html">Quick_Reference</a><br />
<a href="http://kramdown.rubyforge.org/syntax.html">Syntax</a></p>

<p>Features of origin <a href="http://daringfireball.net/projects/markdown/syntax">Markdown</a><a href="http://wowubuntu.com/markdown/basic.html">中文</a></p>

<ul>
  <li>Head. use<code>====</code>or<code>----</code>under head.or use<code>#</code>or<code>##</code> or <code>###</code> or <code>####</code> … before head</li>
  <li>Quote. use <code>&gt;</code> infront of quote brock</li>
  <li>Emphasize. use <code>*</code> or <code>**</code> or <code>_</code> or <code>__</code> wrap text</li>
  <li>List. use <code>* item</code>,<code>+ item</code>,<code>- item</code> or <code>1. item</code>,<code>2. item</code>,<code>3. item</code>start the line</li>
  <li>Link. use <code>[example](link)</code> or <code>[example][index]</code> and put <code>[index]:link</code>in the
end</li>
  <li>Image. same as link, but put <code>!</code> in the front</li>
  <li>Code in line. use <code>`code`</code> </li>
  <li>Code in block. use 4 space in front</li>
</ul>

<p>Attracting features:</p>

<ul>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#footnotes">Footnote</a>. use <code>[^1]</code> in line and <code>[^1]</code> in the end.</li>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#abbreviations">Abbreviation</a>. use <code>*[term]: defination</code> in the end.</li>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#tables">Table</a>. same as <abbr title="reStructuredText">rst</abbr></li>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#inline-attributes">Span Style</a>. use <code>red{:style:"color:red"}</code> inline behind the term.</li>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#code-blocks">Code block</a>. use <code>~~~~</code> start, use longer <code>~~~~~~~</code> end.</li>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#horizontal-rules">Horizontal Rules</a>. use <code>* * * *</code> or <code>- - - -</code> or <code>--------</code></li>
  <li><a href="http://kramdown.rubyforge.org/syntax.html#math-blocks">LaTeX</a>. use <code>$$</code> start, <code>$$</code> end</li>
</ul>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/13/evolution-seminar/">Evolution Seminar</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-13T12:04:00-05:00" pubdate data-updated="true">Sep 13<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p>by David</p>

<p>Biodiversity genetics:</p>

<ul>
  <li>wingsize in nasonia</li>
  <li>pheromones in Drosophlia</li>
  <li>Repeated evolution of alcohol consuption</li>
</ul>

<h4 id="mainly-about-the-first-one">mainly about the first one:</h4>

<p>Nasonia: parasitoid wasp, Haplo-diploid genetics</p>

<p><strong>Wing size: rapid evolution of sex-special</strong></p>

<p>What genes are invloved in growth?</p>

<ul>
  <li>down stream: founction genes</li>
  <li>up stream: regulatory genes</li>
</ul>

<ol>
  <li>hybrid backcross</li>
  <li>wing QTL introgerssion lines </li>
  <li>linked mutant markers</li>
  <li>fine scale recombinants</li>
</ol>

<p>recombination mapping wdw to clone genes
cis-regulation: could be adjacent genes.
dsx^M expression is corrletated with wing size</p>

<p>upd: Regulatory signaling in the JAK/STAT pathway
wdw drive upd-like expression</p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/12/genetic-colloquium-jimingjiang/">Genetic_Colloquium_JimingJiang</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-12T15:32:00-05:00" pubdate data-updated="true">Sep 12<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><a href="http://en.wikipedia.org/wiki/Centromere">centromeres</a>
<a href="http://en.wikipedia.org/wiki/Kinetochore">Kinetochore</a>
<a href="http://en.wikipedia.org/wiki/Microtubule">Microtubule</a></p>

<p>centromere: histone CENH3 -&gt; H3</p>

<ul>
  <li>Heterochromatin in Srosophlia melanogaster: 420kb</li>
  <li>The C-banding patterns of human chromosome: 0.25~5Mb a satellite</li>
</ul>

<p>Centromeres are evolutionary junkyards of eukaryotic genomes:</p>

<ol>
  <li>Centomeres in most higher eukaryotic … </li>
  <li>do not contain genes </li>
  <li>cross over is completely suppressed </li>
</ol>

<p>The brith and death of centromeres:</p>

<ul>
  <li>joint</li>
  <li>moved from satellite, old still there</li>
</ul>

<p>sentromere repositioning during evolution:</p>

<ul>
  <li>no rearranging. </li>
</ul>

<p>Neocentromere-mediateed centromere evolution:</p>

<ul>
  <li>somthing in between?</li>
</ul>

<p>~~~~~~~~~~~~~~~~Work~~~~~~~~~~~~~~~~~</p>

<p>CentO satellite repeat in rice replace:</p>

<ul>
  <li>range from 60kb - 4Mb</li>
</ul>

<p>potato centromeres:</p>

<ul>
  <li>sequencing reads mapping peak</li>
  <li>those with no peak: satellate repeat 1.5Mb</li>
</ul>

<p>The starnge patato centormeres:
1. 6 satellites rep unique
2. 5 centormeres dose not contain satellites repeat
3. one centormere contain both …</p>

<p>An evolutionary new centromere will have battle with the original centeomere in
the populations</p>

<p>Different haplotypes of Potato centromere 2:</p>

<ul>
  <li>Atzimba VS Kennebec VS Kerr’s Pink</li>
</ul>

<p>How an evolutionary new centromere eventually prevails?</p>

<ul>
  <li>Human neroecntromeres are small(90-469kb) VS normal centromers(540-1700kb)</li>
  <li>Cen2 in Oats(4Mb) &gt; Cen2 in Maize(B73)(2Mb): expanded in one direction @ same
place</li>
  <li>CENH3 nucleosomes appear to be incompatible to transcription -&gt; No genes.</li>
  <li>alter the transcription</li>
  <li>may impact the recombination of many genes in the centromeric regions</li>
  <li>may impact the epigenetic modification</li>
</ul>

<p>take home
1. centromeres evolved from neocentromeres
2. evolve slowly or change from “repeatless” pattern to “repeat-based” pattern
   in a sudden fasion
3. Neocentromere activation will affect transcription and recombintation of
   gene located in teh centromeric and pericentromeric regions.</p>

<p>if a centromeres longer than 10Mb it’s not stable.</p>
</div>
  
  


    </article>
  
  <div class="pagination">
    
      <a class="prev" href="/blog/page/2/">&larr; Older</a>
    
    <a href="/blog/archives">Blog Archives</a>
    
  </div>
</div>
<aside class="sidebar">
  
    <section>
  <h1>Recent Posts</h1>
  <ul id="recent_posts">
    
      <li class="post">
        <a href="/blog/2012/09/24/structural-functional-annotation/">structural functional annotation</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/20/biochem-note-1/">biochem note 1</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/19/genome-alignment-and-rna-seq/">Genome alignment and RNA-seq</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/18/colorful-python/">让你的python亮起来</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/17/variant-discovery-using-ngs/">Variant discovery using NGS</a>
      </li>
    
  </ul>
</section>

<section>
  <h1>GitHub Repos</h1>
  <ul id="gh_repos">
    <li class="loading">Status updating&#8230;</li>
  </ul>
  
  <a href="https://github.com/bingw">@bingw</a> on GitHub
  
  <script type="text/javascript">
    $.domReady(function(){
        if (!window.jXHR){
            var jxhr = document.createElement('script');
            jxhr.type = 'text/javascript';
            jxhr.src = '/javascripts/libs/jXHR.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(jxhr, s);
        }

        github.showRepos({
            user: 'bingw',
            count: 0,
            skip_forks: true,
            target: '#gh_repos'
        });
    });
  </script>
  <script src="/javascripts/github.js" type="text/javascript"> </script>
</section>






  
</aside>

    </div>
  </div>
  <footer role="contentinfo"><p>
  Copyright &copy; 2012 - BingW -
  <span class="credit">Powered by <a href="http://octopress.org">Octopress</a></span>
</p>

</footer>
  











</body>
</html>
