
<!DOCTYPE html>
<!--[if IEMobile 7 ]><html class="no-js iem7"><![endif]-->
<!--[if lt IE 9]><html class="no-js lte-ie8"><![endif]-->
<!--[if (gt IE 8)|(gt IEMobile 7)|!(IEMobile)|!(IE)]><!--><html class="no-js" lang="en"><!--<![endif]-->
<head>
  <meta charset="utf-8">
  <title>Bing & KK</title>
  <meta name="author" content="BingW">

  
  <meta name="description" content="A frame work for variation discovery and genotyping using next-genetation DNA sequencing data
Mark A DePristo .etc Abstract
A frame work to discovery &hellip;">
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://BingW.github.com/">
  <link href="/favicon.png" rel="icon">
  <link href="/stylesheets/screen.css" media="screen, projection" rel="stylesheet" type="text/css">
  <script src="/javascripts/modernizr-2.0.js"></script>
  <script src="/javascripts/ender.js"></script>
  <script src="/javascripts/octopress.js" type="text/javascript"></script>
  <link href="/atom.xml" rel="alternate" title="Bing & KK" type="application/atom+xml">
  <!--Fonts from Google"s Web font directory at http://google.com/webfonts -->
<link href="http://fonts.googleapis.com/css?family=PT+Serif:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">

<!-- mathjax config similar to math.stackexchange -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
      }
    });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Queue(function() {
        var all = MathJax.Hub.getAllJax(), i;
        for(i=0; i < all.length; i += 1) {
            all[i].SourceElement().parentNode.className += ' has-jax';
        }
    });
</script>

<script type="text/javascript"
   src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>

  

</head>

<body   >
  <header role="banner"><hgroup>
  <h1><a href="/">Bing & KK</a></h1>
  
    <h2>Conding with Zen</h2>
  
</hgroup>

</header>
  <nav role="navigation"><ul class="subscription" data-subscription="rss">
  <li><a href="/atom.xml" rel="subscribe-rss" title="subscribe via RSS">RSS</a></li>
  
</ul>
  
<form action="http://google.com/search" method="get">
  <fieldset role="search">
    <input type="hidden" name="q" value="site:BingW.github.com" />
    <input class="search" type="text" name="q" results="0" placeholder="Search"/>
  </fieldset>
</form>
  
<ul class="main-navigation">
  <li><a href="/">Blog</a></li>
  <li><a href="/blog/archives">Archives</a></li>
</ul>

</nav>
  <div id="main">
    <div id="content">
      <div class="blog-index">
  
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/17/depristo-variationdiscovery/">DePristo_VariationDiscovery</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-17T00:26:00-05:00" pubdate data-updated="true">Sep 17<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><h3 id="a-frame-work-for-variation-discovery-and-genotyping-using-next-genetation-dna-sequencing-data">A frame work for variation discovery and genotyping using next-genetation DNA sequencing data</h3>
<p><em>Mark A DePristo .etc</em></p>

<h4 id="abstract">Abstract</h4>
<p>A frame work to discovery and genotype variation among multiple samples <br />
across five sequencing technologies and three canonical experimental design.</p>

<p>process:</p>

<ul>
  <li>initial read mapping</li>
  <li>local alignment around indels</li>
  <li>base quality score recalibration</li>
  <li>SNP discovery and genotyping to find all potential variants</li>
  <li>machine learing to separate true segreating variation from machnie artifacts
common to next-generation sequencing tech.</li>
</ul>

<h4 id="introduction">Introduction</h4>
<p>three datasets:
<img src="../images/DePristo_table1.png" alt="NGS datasets analyzed" /></p>

<h4 id="results">Results</h4>
<p><img src="../images/DePristo_fig1.png" alt="Three-part conceptual framework" /></p>

<p>instantiated in Genome Analysis Toolkit <a href="http://genome.cshlp.org/content/20/9/1297"><abbr title="Genome Analysis Toolkit">GATK</abbr></a></p>

<h4 id="applying-the-analysis-pipeline-to-hiseq28-mb-exome-capturelow-pass-4x-sequencing">Applying the analysis pipeline to Hiseq/28-Mb exome capture/low-pass (4x) sequencing</h4>
<p><img src="../images/DePristo_table2.png" alt="Raw to racalibrated" /></p>

<p><img src="../images/DePristo_fig4.png" alt="Results of variant quality recalibration" /></p>

<p>Some concepts:<abbr title="transition(A<->G,T<->C) / transversion(others..)&#8221;>Ti/Tv</abbr>,<abbr title="linkage disequilibrium">LD</abbr>,<abbr title="copy number variations">CNVs</abbr>,<abbr title="next-generation sequencing">NGS</abbr>,<abbr title="1000 Genomes Project">1KG</abbr>,<abbr title="Genome Analysis Toolkit">GATK</abbr>,<abbr title="false discovery rate">FDR</abbr>,<abbr title="Single Nucleotide Polymorphism Database">dbSNP</abbr>,<abbr title="whole genome shotgun">WGS</abbr>,<abbr title="non-reference discrepancy">NRD</abbr>,<abbr title="non-reference sensitivity">NR</abbr>,<abbr title="multiple sequence realignment">MSA</abbr>,<abbr title="Root-mean-square errors">RMSE</abbr>,<abbr title="HapMap3">HM3</abbr></p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/13/yaml-format/">YAML Format</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-13T16:39:00-05:00" pubdate data-updated="true">Sep 13<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p>start with <code>---</code> end with <code>---</code> at the head of your document:</p>

<ul>
  <li>layout: post</li>
  <li>title: “Hello World”</li>
  <li>date: 2012-06-19 05:20 </li>
  <li>comments: true </li>
  <li>categories: test</li>
  <li>author: Bing Wang</li>
  <li>
    <p>published: true</p>
  </li>
  <li>external_url: </li>
  <li>sharing: true</li>
  <li>footer: true</li>
</ul>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/13/kramdown-syntax/">Kramdown Syntax</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-13T14:42:00-05:00" pubdate data-updated="true">Sep 13<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><a href="http://kramdown.rubyforge.org/quickref.html">Quick_Reference</a><br />
<a href="http://kramdown.rubyforge.org/syntax.html">Syntax</a></p>

<p>Features of origin <a href="http://daringfireball.net/projects/markdown/syntax">Markdown</a><a href="http://wowubuntu.com/markdown/basic.html">中文</a></p>

<ul>
  <li>Head. use<code>====</code>or<code>----</code>under head.or use<code>#</code>or<code>##</code> or <code>###</code> or <code>####</code> … before head</li>
  <li>Quote. use <code>&gt;</code> infront of quote brock</li>
  <li>Emphasize. use <code>*</code> or <code>**</code> or <code>_</code> or <code>__</code> wrap text</li>
  <li>List. use <code>* item</code>,<code>+ item</code>,<code>- item</code> or <code>1. item</code>,<code>2. item</code>,<code>3. item</code>start the line</li>
  <li>Link. use <code>[example](link)</code> or <code>[example][index]</code> and put <code>[index]:link</code>in the
end</li>
  <li>Image. same as link, but put <code>!</code> in the front</li>
  <li>Code in line. use <code>`code`</code> </li>
  <li>Code in block. use 4 space in front</li>
</ul>

<p>Attracting features:</p>

<ul>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#footnotes">Footnote</a>. use <code>[^1]</code> in line and <code>[^1]</code> in the end.</li>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#abbreviations">Abbreviation</a>. use <code>*[term]: defination</code> in the end.</li>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#tables">Table</a>. same as <abbr title="reStructuredText">rst</abbr></li>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#inline-attributes">Span Style</a>. use <code>red{:style:"color:red"}</code> inline behind the term.</li>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#code-blocks">Code block</a>. use <code>~~~~</code> start, use longer <code>~~~~~~~</code> end.</li>
  <li><a href="http://kramdown.rubyforge.org/quickref.html#horizontal-rules">Horizontal Rules</a>. use <code>* * * *</code> or <code>- - - -</code> or <code>--------</code></li>
  <li><a href="http://kramdown.rubyforge.org/syntax.html#math-blocks">LaTeX</a>. use <code>$$</code> start, <code>$$</code> end</li>
</ul>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/13/evolution-seminar/">Evolution Seminar</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-13T12:04:00-05:00" pubdate data-updated="true">Sep 13<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p>by David</p>

<p>Biodiversity genetics:</p>

<ul>
  <li>wingsize in nasonia</li>
  <li>pheromones in Drosophlia</li>
  <li>Repeated evolution of alcohol consuption</li>
</ul>

<h4 id="mainly-about-the-first-one">mainly about the first one:</h4>

<p>Nasonia: parasitoid wasp, Haplo-diploid genetics</p>

<p><strong>Wing size: rapid evolution of sex-special</strong></p>

<p>What genes are invloved in growth?</p>

<ul>
  <li>down stream: founction genes</li>
  <li>up stream: regulatory genes</li>
</ul>

<ol>
  <li>hybrid backcross</li>
  <li>wing QTL introgerssion lines </li>
  <li>linked mutant markers</li>
  <li>fine scale recombinants</li>
</ol>

<p>recombination mapping wdw to clone genes
cis-regulation: could be adjacent genes.
dsx^M expression is corrletated with wing size</p>

<p>upd: Regulatory signaling in the JAK/STAT pathway
wdw drive upd-like expression</p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/12/genetic-colloquium-jimingjiang/">Genetic_Colloquium_JimingJiang</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-12T15:32:00-05:00" pubdate data-updated="true">Sep 12<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p><a href="http://en.wikipedia.org/wiki/Centromere">centromeres</a>
<a href="http://en.wikipedia.org/wiki/Kinetochore">Kinetochore</a>
<a href="http://en.wikipedia.org/wiki/Microtubule">Microtubule</a></p>

<p>centromere: histone CENH3 -&gt; H3</p>

<ul>
  <li>Heterochromatin in Srosophlia melanogaster: 420kb</li>
  <li>The C-banding patterns of human chromosome: 0.25~5Mb a satellite</li>
</ul>

<p>Centromeres are evolutionary junkyards of eukaryotic genomes:</p>

<ol>
  <li>Centomeres in most higher eukaryotic … </li>
  <li>do not contain genes </li>
  <li>cross over is completely suppressed </li>
</ol>

<p>The brith and death of centromeres:</p>

<ul>
  <li>joint</li>
  <li>moved from satellite, old still there</li>
</ul>

<p>sentromere repositioning during evolution:</p>

<ul>
  <li>no rearranging. </li>
</ul>

<p>Neocentromere-mediateed centromere evolution:</p>

<ul>
  <li>somthing in between?</li>
</ul>

<p>~~~~~~~~~~~~~~~~Work~~~~~~~~~~~~~~~~~</p>

<p>CentO satellite repeat in rice replace:</p>

<ul>
  <li>range from 60kb - 4Mb</li>
</ul>

<p>potato centromeres:</p>

<ul>
  <li>sequencing reads mapping peak</li>
  <li>those with no peak: satellate repeat 1.5Mb</li>
</ul>

<p>The starnge patato centormeres:
1. 6 satellites rep unique
2. 5 centormeres dose not contain satellites repeat
3. one centormere contain both …</p>

<p>An evolutionary new centromere will have battle with the original centeomere in
the populations</p>

<p>Different haplotypes of Potato centromere 2:</p>

<ul>
  <li>Atzimba VS Kennebec VS Kerr’s Pink</li>
</ul>

<p>How an evolutionary new centromere eventually prevails?</p>

<ul>
  <li>Human neroecntromeres are small(90-469kb) VS normal centromers(540-1700kb)</li>
  <li>Cen2 in Oats(4Mb) &gt; Cen2 in Maize(B73)(2Mb): expanded in one direction @ same
place</li>
  <li>CENH3 nucleosomes appear to be incompatible to transcription -&gt; No genes.</li>
  <li>alter the transcription</li>
  <li>may impact the recombination of many genes in the centromeric regions</li>
  <li>may impact the epigenetic modification</li>
</ul>

<p>take home
1. centromeres evolved from neocentromeres
2. evolve slowly or change from “repeatless” pattern to “repeat-based” pattern
   in a sudden fasion
3. Neocentromere activation will affect transcription and recombintation of
   gene located in teh centromeric and pericentromeric regions.</p>

<p>if a centromeres longer than 10Mb it’s not stable.</p>
</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/12/short-read-mapping/">Short-read Mapping</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-12T14:27:00-05:00" pubdate data-updated="true">Sep 12<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><p>concept: TP FP FN TN</p>

<p>When does alignment accuracy matter? tradeoff bettwen speed and accuracy.</p>

<h4 id="bw-algorithms">BW-algorithms</h4>
<p><img src="http://bingw.github.com/images/BW.png" alt="figure1" /></p>

<h4 id="algorithms-and-speed-up">Algorithms and speed up</h4>

<ul>
  <li>use alignment refinement</li>
</ul>

<p><a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml">Bowtie 2</a></p>

<h4 id="dynamic-programming">Dynamic Programming</h4>
<p><a href="http://en.wikipedia.org/wiki/Smith%E2%80%93Waterman_algorithm">Smith-Waterman</a></p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/10/short-read-sequencing/">Short-read Sequencing</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-10T14:23:00-05:00" pubdate data-updated="true">Sep 10<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><h3 id="sequencing-and-de-novo-assembly">Sequencing and de novo assembly</h3>

<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/21307932">Sequencing technology</a>
<img src="http://bingw.github.com/images/short_read_sequencing_t1.png" alt="Sequencing technology" /></p>

<p>concept: mate pair VS paired end</p>

<h4 id="the-process">The Process:</h4>
<ol>
  <li>Choose sequencing platform</li>
  <li>Make DNA</li>
  <li>Prepare Library</li>
  <li>sequence Run</li>
  <li>primary Analysis Pipeline
 Converts image files (Tb) to sequences (100’s Gb).(inreversbale)</li>
  <li>Obtain Sequence Reads</li>
</ol>

<h4 id="sequence-quality">Sequence quality:</h4>

<p>Phred Score VS accuracy:</p>

<ul>
  <li>10 ==&gt; 90%</li>
  <li>20 ==&gt; 99%</li>
  <li>30 ==&gt; 99.9%</li>
  <li>40 ==&gt; 99.99%</li>
  <li>50 ==&gt; 99.999%</li>
</ul>

<h4 id="format">Format:</h4>

<p>different identifier for phred<a href="http://en.wikipedia.org/wiki/FASTQ_format">wiki</a></p>

<p>A software for this<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">FastQC</a></p>

<h4 id="genome-scale-sequence-analysis">Genome-scale Sequence Analysis</h4>

<ul>
  <li><strong>De nove assembly</strong></li>
  <li>Templated assembly</li>
  <li>Read mapping or alignment to a reference genome</li>
</ul>

<h4 id="assemblers">Assemblers</h4>

<p>overlap graph assemblers:</p>

<ul>
  <li>Screen out known repeats and set them aside for later</li>
  <li>Infer repetitiveness based on coverage</li>
  <li>First assemble unambiguous overlaps, then resolve repeats using mate pairs</li>
</ul>

<p>De Bruijn K-mers assemblers problems:</p>

<ul>
  <li>Require large amount of memory to store graph</li>
  <li>Not as easy to parallelize as overlap assemblers</li>
</ul>

<h4 id="how-many-clonesreads-do-we-need">How many clones/reads do we need?</h4>

<ul>
  <li>G = Genome Length</li>
  <li>L = Sequence Read Length</li>
  <li>N = Number of Sequences Collected</li>
  <li>T = Number of Basepairs of Overlap Needed</li>
</ul>

<p>Number of Contigs = $N \cdot e^{-\frac{LN}{G} \cdot (1-\frac{T}{L})}$
(<a href="http://www.ncbi.nlm.nih.gov/pubmed/3294162">LW prediction</a>)</p>

<h4 id="gap-closure-strategies">Gap Closure Strategies</h4>

<ul>
  <li>Primer walk to sequence the rest of linking clones that span a scaffold gap</li>
  <li>Primer walk off clones at the ends of contigs for which there is no linking information</li>
  <li>PCR based on your best guess at contig order (comparison to other closely related genomes, predicted genes at the end of genomes, anything else you can come up with)</li>
  <li>Combinatorial PCR with primers designed at the end of each contig</li>
</ul>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/10/schatz2010-assemblyreview/">Schatz2010_AssemblyReview</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-10T12:13:00-05:00" pubdate data-updated="true">Sep 10<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><h3 id="assembly-of-large-genomes-using-second-generation-sequencing">Assembly of large genomes using second-generation sequencing</h3>

<p><em>Michael C. Schatz, Arthur L. Delcher and Steven L. Salzberg</em></p>

</div>
  
  


    </article>
  
  
    <article>
      
  <header>
    
      <h1 class="entry-title"><a href="/blog/2012/09/09/FlicekBirney2009-NGSReview/">FlicekBirney2009-NGSReview</a></h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-09T14:02:00-05:00" pubdate data-updated="true">Sep 9<span>th</span>, 2012</time>
        
      </p>
    
  </header>


  <div class="entry-content"><h3 id="sense-from-sequence-reads-methods-for-alignment-and-assembly">Sense From Sequence Reads: Methods for Alignment and Assembly</h3>

<p><em>Paul Flicek &amp; Ewan Birney</em></p>

<h4 id="abstruct">Abstruct</h4>

<p>review of alignment and assembly algorithms</p>

<h4 id="background">Background</h4>

<p>quality versus cost tradeoff:</p>

<ul>
  <li>de novo sequencing: longer reads</li>
  <li>resequencing: high coverage, paired-end reads</li>
  <li>mapping TF binding site/location of modified histones: <a href="http://en.wikipedia.org/wiki/Chromatin_immunoprecipitation">ChIP</a>, short reads</li>
  <li>DNase-seq: limited to only 20bp of informative sequence</li>
</ul>

<p>Different error characteristics: </p>

<ul>
  <li>Roche 454:insertion or deletion errors during homopolymer runs</li>
  <li>ABI SOLiD &amp; Illumina/Solexa: errors toward the end of the read</li>
</ul>

<h4 id="alignment">Alignment</h4>

<p>Alignment methods(tradeoff between speed and sensitivity):</p>

<ul>
  <li>hash table-based implementations</li>
  <li>Burrows Wheeler transform(BWT)-based methods</li>
</ul>

<p>Alignment procedure:</p>

<ol>
  <li>heuristic techniques</li>
  <li>accurate alignment alogrithms(Smith-Waterman)</li>
</ol>

<h5 id="hash-based-alignment-methods">Hash-based alignment methods</h5>

<p>Sepecially for short reads. methods include:</p>

<ul>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18714091&amp;dopt=Abstract">MAQ</a>:
  From the first 28 bp of the read, MAQ program builds 6 hash tables corresponding 
  to seeds of (length = 8, weight = 4). scans the reference genome against these 
  hash tables. This ensures that all hits with 2 mismatches can be found. 20 hash 
  tables required if get all 3 mismatches.</li>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18227114&amp;dopt=Abstract">SOPA</a></li>
  <li>Illumina’s unpublished ELAND algorithm</li>
</ul>

<p>These have recently been joined by:</p>

<ul>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=19461883&amp;dopt=Abstract">SHRiMP</a>:
  use a <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16597241&amp;dopt=Abstract">q-gram</a> approach, (require mutiple sapced seeds per read</li>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18684737&amp;dopt=Abstract">ZOOM</a>:
  manually constructed spaced seeds of weight 14 to enable the detection of up to 4    mismatches in 50-bp reads</li>
  <li><a href="http://genome.ucla.edu/bfast/">BFAST</a></li>
  <li><a href="http://bioinformatics.bc.edu/marthlab/Mosaik/">MOSAIK</a>:
  hashes all positions in the reference genome and uses a ‘jump database’ to 
  efficiently locate information in the hash table and thus reduces memory 
  requirements by approximately two-thirds over a naive implementation.</li>
</ul>

<h5 id="burrows-wheeler-transform-methods">Burrows-Wheeler Transform methods</h5>

<p>use the FM index data structure, retains the suffix array’s ability for rapid subsequence
search. Much faster than their hash-based counterparts at the same sensitivity level.</p>

<ol>
  <li>the sequence order of the reference genome is modified using the BWT(reversible)</li>
  <li>the final index is created; it is then used for rapid read placement on the
genome(memory-intensive)</li>
</ol>

<p>softwares:</p>

<ul>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=19261174&amp;dopt=Abstract">BOWTIE</a>:
  Reported 30-fold speed increase over hash-based MAQ</li>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=19451168&amp;dopt=Abstract">BWA</a></li>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=19497933&amp;dopt=Abstract">SOAP2</a></li>
</ul>

<h4 id="assembly">Assembly</h4>

<p><img src="http://bingw.github.com/images/paul_alignment.png" alt="de Bruijn graph" /></p>

<ol>
  <li>Sequence at top represents the genome, which is then sampled using shotgun sequencing in base space with 7-bp reads.</li>
  <li>the k-mers in the reads (4-mers in this example) are collected into nodes and the coverage at each node is recorded.</li>
  <li>he graph is simplified to combine nodes that are associated with the continuous linear stretches into single, larger nodes of various k-mer sizes.</li>
  <li>error correction removes the tips and bubbles that result from sequencing errors and creates a final graph structure that accurately and completely describes in the original genome sequence.</li>
</ol>

<p>softwares:</p>

<ul>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17908823&amp;dopt=Abstract">SHARCGS</a></li>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=17893086&amp;dopt=Abstract">VCAKE</a></li>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18349386&amp;dopt=Abstract">VELVET</a></li>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18083777&amp;dopt=Abstract">EULER-SR</a></li>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18332092&amp;dopt=Abstract">EDENA</a></li>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=19251739&amp;dopt=Abstract">ABySS</a></li>
  <li><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=18340039&amp;dopt=Abstract">ALLPATHS</a></li>
</ul>

<h4 id="disscusion">Disscusion</h4>

<p>List of sequence alignment software: <a href="http://en.wikipedia.org/wiki/List_of_sequence_alignment_software">wiki</a>, <a href="http://seqanswers.com/forums/showthread.php?t=43">SEQanswers</a></p>

</div>
  
  


    </article>
  
  <div class="pagination">
    
    <a href="/blog/archives">Blog Archives</a>
    
  </div>
</div>
<aside class="sidebar">
  
    <section>
  <h1>Recent Posts</h1>
  <ul id="recent_posts">
    
      <li class="post">
        <a href="/blog/2012/09/17/depristo-variationdiscovery/">DePristo_VariationDiscovery</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/13/yaml-format/">YAML Format</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/13/kramdown-syntax/">Kramdown Syntax</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/13/evolution-seminar/">evolution seminar</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/12/genetic-colloquium-jimingjiang/">Genetic_Colloquium_JimingJiang</a>
      </li>
    
  </ul>
</section>

<section>
  <h1>GitHub Repos</h1>
  <ul id="gh_repos">
    <li class="loading">Status updating&#8230;</li>
  </ul>
  
  <a href="https://github.com/bingw">@bingw</a> on GitHub
  
  <script type="text/javascript">
    $.domReady(function(){
        if (!window.jXHR){
            var jxhr = document.createElement('script');
            jxhr.type = 'text/javascript';
            jxhr.src = '/javascripts/libs/jXHR.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(jxhr, s);
        }

        github.showRepos({
            user: 'bingw',
            count: 0,
            skip_forks: true,
            target: '#gh_repos'
        });
    });
  </script>
  <script src="/javascripts/github.js" type="text/javascript"> </script>
</section>






  
</aside>

    </div>
  </div>
  <footer role="contentinfo"><p>
  Copyright &copy; 2012 - BingW -
  <span class="credit">Powered by <a href="http://octopress.org">Octopress</a></span>
</p>

</footer>
  











</body>
</html>
