
<!DOCTYPE html>
<!--[if IEMobile 7 ]><html class="no-js iem7"><![endif]-->
<!--[if lt IE 9]><html class="no-js lte-ie8"><![endif]-->
<!--[if (gt IE 8)|(gt IEMobile 7)|!(IEMobile)|!(IE)]><!--><html class="no-js" lang="en"><!--<![endif]-->
<head>
  <meta charset="utf-8">
  <title>Devin's Pipeline - Bing & KK</title>
  <meta name="author" content="BingW">

  
  <meta name="description" content="HMMER 1.8.4 -&gt; detect high scoring matches to (an HMM created from an alignment of S.cerevisiae introns and flanking sequences.) Generated all &hellip;">
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://BingW.github.com/blog/2012/09/11/devin-pipeline/">
  <link href="/favicon.png" rel="icon">
  <link href="/stylesheets/screen.css" media="screen, projection" rel="stylesheet" type="text/css">
  <script src="/javascripts/modernizr-2.0.js"></script>
  <script src="/javascripts/ender.js"></script>
  <script src="/javascripts/octopress.js" type="text/javascript"></script>
  <link href="/atom.xml" rel="alternate" title="Bing & KK" type="application/atom+xml">
  <!--Fonts from Google"s Web font directory at http://google.com/webfonts -->
<link href="http://fonts.googleapis.com/css?family=PT+Serif:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">

<!-- mathjax config similar to math.stackexchange -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
      }
    });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Queue(function() {
        var all = MathJax.Hub.getAllJax(), i;
        for(i=0; i < all.length; i += 1) {
            all[i].SourceElement().parentNode.className += ' has-jax';
        }
    });
</script>

<script type="text/javascript"
   src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>

  

</head>

<body   >
  <header role="banner"><hgroup>
  <h1><a href="/">Bing & KK</a></h1>
  
    <h2>Conding with Zen</h2>
  
</hgroup>

</header>
  <nav role="navigation"><ul class="subscription" data-subscription="rss">
  <li><a href="/atom.xml" rel="subscribe-rss" title="subscribe via RSS">RSS</a></li>
  
</ul>
  
<form action="http://google.com/search" method="get">
  <fieldset role="search">
    <input type="hidden" name="q" value="site:BingW.github.com" />
    <input class="search" type="text" name="q" results="0" placeholder="Search"/>
  </fieldset>
</form>
  
<ul class="main-navigation">
  <li><a href="/">Blog</a></li>
  <li><a href="/blog/archives">Archives</a></li>
</ul>

</nav>
  <div id="main">
    <div id="content">
      <div>
<article class="hentry" role="article">
  
  <header>
    
      <h1 class="entry-title">Devin's Pipeline</h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-11T13:11:00-05:00" pubdate data-updated="true">Sep 11<span>th</span>, 2012</time>
        
      </p>
    
  </header>


<div class="entry-content"><ol>
  <li>
    <p><strong>HMMER 1.8.4</strong> -&gt; detect high scoring matches to 
(an HMM created from an alignment of S.cerevisiae introns and flanking sequences.)</p>
  </li>
  <li>
    <p>Generated all ORFs above a context-dependent minimum size (60 bp at contigs ends and intron 
splice-sites, 300 bp at other locations), combined these with the putative splice-sites and 
selected the most upstream ATG as the putative start codon.</p>
  </li>
  <li>
    <p>Retained only possible gene models longer than 300 bp.</p>
  </li>
  <li>
    <p>Potential ORFs that spanned high-scoring matches to the splice site model were represented 
by multiple alternative gene models.</p>
  </li>
  <li>
    <p>At each locus, we aligned alternative models to the best available homolog (typically from 
S. cerevisiae) using <strong>Exonerate</strong> (Slater and Birney 2005) (-model affine:global -exhaustive 1 -bestn 1)
and selected the model with the highest score.</p>
  </li>
  <li>Sought positive evidence for all remaining gene models by <strong>BLAST</strong> (Altschul et al. 1997) against a 
local database of:
    <ul>
      <li>yeast proteins assembled from completely sequenced yeast genomes (Wood et al. 2002; Cliften et al. 2003
 ; Dujon et al. 2004; Kellis et al. 2004; Souciet et al. 2009), </li>
      <li>the NCBI nr protein database</li>
      <li>S. cerevisiae Ty and LTR sequences downloaded from Saccharomyces Genome Database (Engel et al. 2010) 
 (SGD, http:// www.yeastgenome.org)</li>
    </ul>
  </li>
  <li>
    <p>Use HMMER3 (Eddy 2009) to search our gene models against a database of 4704 protein HMMs derived 
from the Yeast Gene Order Browser (Byrne and Wolfe 2006) (YGOB, http://wolfe.gen.tcd.ie/ygob).</p>
  </li>
  <li>
    <p>In order to recover small genes or rapidly evolving genes, we subjected all presumed intergenic 
regions to low stringency BLAST searches against our local database of yeast proteins and searched
genomic regions predicted by synteny to contain YGOB matches with our YGOB-HMM library.</p>
  </li>
  <li>
    <p>To detect potential novel genes, we used linear discriminant analysis to assign ORFs lacking 
homology to genes in the YGOB or yeast ortholog databases a “coding probability” based on whether 
their codon usage frequencies more closely resembled confirmed protein-coding genes or confirmed 
intergenic regions.</p>
  </li>
  <li>
    <p>Eliminated poorly supported gene models that overlapped well-supported models and merged
neighboring models that were likely to be fragments of the same gene.</p>
  </li>
  <li>
    <p>The fragmented gene models were typically created by scaffold gaps or frameshifts
(either real or due to sequencing errors).</p>
  </li>
  <li>
    <p>The remaining models were stratified according to the quality of the supporting evidence and 
comprised our initial protein-coding gene set.</p>
  </li>
  <li>We used <strong>tRNA-ScanSE</strong> to identify tRNA genes (Lowe and Eddy 1997).</li>
</ol>
</div>


  <footer>
    <p class="meta">
      
  

<span class="byline author vcard">Posted by <span class="fn">BingW</span></span>

      








  


<time datetime="2012-09-11T13:11:00-05:00" pubdate data-updated="true">Sep 11<span>th</span>, 2012</time>
      


    </p>
    
      <div class="sharing">
  
  
  
</div>

    
    <p class="meta">
      
        <a class="basic-alignment left" href="/blog/2012/09/10/short-read-sequencing/" title="Previous Post: Short-read sequencing">&laquo; Short-read sequencing</a>
      
      
        <a class="basic-alignment right" href="/blog/2012/09/12/short-read-mapping/" title="Next Post: Short-read mapping">Short-read mapping &raquo;</a>
      
    </p>
  </footer>
</article>

</div>

<aside class="sidebar">
  
    <section>
  <h1>Recent Posts</h1>
  <ul id="recent_posts">
    
      <li class="post">
        <a href="/blog/2012/09/18/colorful-python/">让你的python亮起来</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/17/variant-discovery-using-ngs/">Variant discovery using NGS</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/17/depristo-variationdiscovery/">DePristo_VariationDiscovery</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/13/yaml-format/">YAML Format</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/13/kramdown-syntax/">Kramdown Syntax</a>
      </li>
    
  </ul>
</section>

<section>
  <h1>GitHub Repos</h1>
  <ul id="gh_repos">
    <li class="loading">Status updating...</li>
  </ul>
  
  <a href="https://github.com/bingw">@bingw</a> on GitHub
  
  <script type="text/javascript">
    $.domReady(function(){
        if (!window.jXHR){
            var jxhr = document.createElement('script');
            jxhr.type = 'text/javascript';
            jxhr.src = '/javascripts/libs/jXHR.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(jxhr, s);
        }

        github.showRepos({
            user: 'bingw',
            count: 0,
            skip_forks: true,
            target: '#gh_repos'
        });
    });
  </script>
  <script src="/javascripts/github.js" type="text/javascript"> </script>
</section>






  
</aside>


    </div>
  </div>
  <footer role="contentinfo"><p>
  Copyright &copy; 2012 - BingW -
  <span class="credit">Powered by <a href="http://octopress.org">Octopress</a></span>
</p>

</footer>
  











</body>
</html>
