
<!DOCTYPE html>
<!--[if IEMobile 7 ]><html class="no-js iem7"><![endif]-->
<!--[if lt IE 9]><html class="no-js lte-ie8"><![endif]-->
<!--[if (gt IE 8)|(gt IEMobile 7)|!(IEMobile)|!(IE)]><!--><html class="no-js" lang="en"><!--<![endif]-->
<head>
  <meta charset="utf-8">
  <title>Genome alignment and RNA-seq - Bing & KK</title>
  <meta name="author" content="BingW">

  
  <meta name="description" content="Genome Alignment Pair-wise Align two genomes Example: MUMmer Multiple or complex samples and a reference genome All of one genome plus whatever parts &hellip;">
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://BingW.github.com/blog/2012/09/19/genome-alignment-and-rna-seq/">
  <link href="/favicon.png" rel="icon">
  <link href="/stylesheets/screen.css" media="screen, projection" rel="stylesheet" type="text/css">
  <script src="/javascripts/modernizr-2.0.js"></script>
  <script src="/javascripts/ender.js"></script>
  <script src="/javascripts/octopress.js" type="text/javascript"></script>
  <link href="/atom.xml" rel="alternate" title="Bing & KK" type="application/atom+xml">
  <!--Fonts from Google"s Web font directory at http://google.com/webfonts -->
<link href="http://fonts.googleapis.com/css?family=PT+Serif:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">

<!-- mathjax config similar to math.stackexchange -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
      }
    });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Queue(function() {
        var all = MathJax.Hub.getAllJax(), i;
        for(i=0; i < all.length; i += 1) {
            all[i].SourceElement().parentNode.className += ' has-jax';
        }
    });
</script>

<script type="text/javascript"
   src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>

  

</head>

<body   >
  <header role="banner"><hgroup>
  <h1><a href="/">Bing & KK</a></h1>
  
    <h2>Conding with Zen</h2>
  
</hgroup>

</header>
  <nav role="navigation"><ul class="subscription" data-subscription="rss">
  <li><a href="/atom.xml" rel="subscribe-rss" title="subscribe via RSS">RSS</a></li>
  
</ul>
  
<form action="http://google.com/search" method="get">
  <fieldset role="search">
    <input type="hidden" name="q" value="site:BingW.github.com" />
    <input class="search" type="text" name="q" results="0" placeholder="Search"/>
  </fieldset>
</form>
  
<ul class="main-navigation">
  <li><a href="/">Blog</a></li>
  <li><a href="/blog/archives">Archives</a></li>
</ul>

</nav>
  <div id="main">
    <div id="content">
      <div>
<article class="hentry" role="article">
  
  <header>
    
      <h1 class="entry-title">Genome Alignment and RNA-seq</h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-19T14:26:00-05:00" pubdate data-updated="true">Sep 19<span>th</span>, 2012</time>
        
      </p>
    
  </header>


<div class="entry-content"><h4 id="genome-alignment">Genome Alignment</h4>
<ul>
  <li>Pair-wise
    <ul>
      <li>Align two genomes</li>
      <li>Example: MUMmer</li>
    </ul>
  </li>
  <li>Multiple or complex samples and a reference genome
    <ul>
      <li>All of one genome plus whatever parts match from the other  genome(s)</li>
      <li>Example: PIPs</li>
    </ul>
  </li>
  <li>Multiple alignment
    <ul>
      <li>All of all the genomes</li>
      <li>Example:  Mauve</li>
    </ul>
  </li>
</ul>

<h5 id="mummerhttpmummersourceforgenet"><a href="http://mummer.sourceforge.net/">MUMmer</a></h5>

<p>Fast pair-wise comparison of draft or complete 
genomes using nucleotide or 6-frame translated 
sequences</p>

<p>? Suffix Tree</p>

<h5 id="percent-identity-plotpip">Percent Identity Plot(PIP)</h5>

<p>quality draft</p>

<h5 id="mauve-multiple-genome-aligner">Mauve multiple Genome Aligner</h5>

<p>Progressive Mauve alignment</p>

<h4 id="rna-seq-goals-and-computational-challenges">RNA-seq Goals and Computational Challenges</h4>

<p>Goal:
+ Annotation
+ Quantification of genes
+ Quantification of isoforms</p>

<p>Challenges:
+ Read mapping
+ Transcriptome reconstruction
+ Expression quantification</p>

<h5 id="domain-specific-challenges">Domain-specific challenges</h5>

<ul>
  <li>Eukaryotes
    <ul>
      <li>Processed and unprocessed RNAs in the sample</li>
      <li>Quality of genome sequences</li>
      <li>Quality of gene (and alternate splicing) annotations</li>
      <li>Isoforms</li>
    </ul>
  </li>
  <li>Prokaryotes
    <ul>
      <li>rRNA</li>
      <li>Gene density</li>
      <li>Operons</li>
    </ul>
  </li>
</ul>

<h5 id="popular-in-wisc">popular in WISC</h5>

<ul>
  <li>Bowtie, Tophat, Cufflinks, CuffDiff</li>
  <li>RSEM, EBSeq</li>
</ul>

<h5 id="read-mapping">Read mapping</h5>

<ul>
  <li>Unspliced aligners: 
    <ul>
      <li>Seed based (Maq, Stampy)</li>
      <li>Burrows Wheeler based (BWA, Bowtie, etc.)</li>
      <li>How similar is your reference genome?</li>
      <li>Best for mapping against cDNA databases from well characterized reference sequences</li>
      <li>Can only recognize known exons and isoforms</li>
    </ul>
  </li>
  <li>Spliced aligners:
    <ul>
      <li>Exon first (MapSplice, SpliceMap, TopHat)
        <ul>
          <li>unspliced alignment, split reads and repeat</li>
          <li>Most efficient if relatively few reads span junctions</li>
          <li>Can miss splice sites if reads map elsewhere in their entirety
 – Seed and Extend (GSNAP, QPALMA)</li>
          <li>Split all reads into seeds, map, merge and extend</li>
          <li>Slower, but detect more splice junctions</li>
          <li>Superior with polymorphic species</li>
        </ul>
      </li>
    </ul>
  </li>
</ul>

<h5 id="transcriptome-reconstruction-challenges">Transcriptome Reconstruction Challenges</h5>

<ul>
  <li>Genes vary in expression level, some may be represented by very few reads</li>
  <li>Reads can come from either mature mRNA or a precursor</li>
  <li>When there are multiple isoforms, how can you tell which gave rise to a particular read?  </li>
</ul>

<h5 id="genome-guided-vs-genome-independent-reconstruction">Genome-guided vs. Genome Independent Reconstruction</h5>

<ul>
  <li>Genome-guided reconstruction
    <ul>
      <li>Exon identification (G.mor.se)</li>
      <li>Genome-guided assembly graph (CuffLinks, Scripture)</li>
    </ul>
  </li>
  <li>Performance varies with the expression level</li>
</ul>

<h5 id="qunantifcation-overview">Qunantifcation overview</h5>

<ul>
  <li><strong>TODO</strong> fig</li>
</ul>

<h5 id="normalization">Normalization</h5>

<ul>
  <li>Genes vary in length - with random fragmentation, long genes are expected to have more reads than short genes expressed at the same level</li>
  <li>Sequence “runs”/ experiments differ in total number of reads collected</li>
  <li>RPKM = Reads per kilobase of transcript per million mapped reads (FPKM = fragments…)</li>
  <li>Many other more sophisticated normalization methods</li>
</ul>

<p>`</p>
</div>


  <footer>
    <p class="meta">
      
  

<span class="byline author vcard">Posted by <span class="fn">BingW</span></span>

      








  


<time datetime="2012-09-19T14:26:00-05:00" pubdate data-updated="true">Sep 19<span>th</span>, 2012</time>
      

<span class="categories">
  
    <a class='category' href='/blog/categories/bioinfo/'>Bioinfo</a>, <a class='category' href='/blog/categories/gen875/'>Gen875</a>
  
</span>


    </p>
    
      <div class="sharing">
  
  
  
</div>

    
    <p class="meta">
      
        <a class="basic-alignment left" href="/blog/2012/09/18/colorful-python/" title="Previous Post: 让你的python亮起来">&laquo; 让你的python亮起来</a>
      
      
        <a class="basic-alignment right" href="/blog/2012/09/20/biochem-note-1/" title="Next Post: biochem note 1">biochem note 1 &raquo;</a>
      
    </p>
  </footer>
</article>

</div>

<aside class="sidebar">
  
    <section>
  <h1>Recent Posts</h1>
  <ul id="recent_posts">
    
      <li class="post">
        <a href="/blog/2012/09/24/structural-functional-annotation/">structural functional annotation</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/20/biochem-note-1/">biochem note 1</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/19/genome-alignment-and-rna-seq/">Genome alignment and RNA-seq</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/18/colorful-python/">让你的python亮起来</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/17/variant-discovery-using-ngs/">Variant discovery using NGS</a>
      </li>
    
  </ul>
</section>

<section>
  <h1>GitHub Repos</h1>
  <ul id="gh_repos">
    <li class="loading">Status updating...</li>
  </ul>
  
  <a href="https://github.com/bingw">@bingw</a> on GitHub
  
  <script type="text/javascript">
    $.domReady(function(){
        if (!window.jXHR){
            var jxhr = document.createElement('script');
            jxhr.type = 'text/javascript';
            jxhr.src = '/javascripts/libs/jXHR.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(jxhr, s);
        }

        github.showRepos({
            user: 'bingw',
            count: 0,
            skip_forks: true,
            target: '#gh_repos'
        });
    });
  </script>
  <script src="/javascripts/github.js" type="text/javascript"> </script>
</section>






  
</aside>


    </div>
  </div>
  <footer role="contentinfo"><p>
  Copyright &copy; 2012 - BingW -
  <span class="credit">Powered by <a href="http://octopress.org">Octopress</a></span>
</p>

</footer>
  











</body>
</html>
