
<!DOCTYPE html>
<!--[if IEMobile 7 ]><html class="no-js iem7"><![endif]-->
<!--[if lt IE 9]><html class="no-js lte-ie8"><![endif]-->
<!--[if (gt IE 8)|(gt IEMobile 7)|!(IEMobile)|!(IE)]><!--><html class="no-js" lang="en"><!--<![endif]-->
<head>
  <meta charset="utf-8">
  <title>Short-read sequencing - Bing & KK</title>
  <meta name="author" content="BingW">

  
  <meta name="description" content="Sequencing and de novo assembly Sequencing technology concept: mate pair VS paired end The Process: Choose sequencing platform Make DNA Prepare &hellip;">
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://BingW.github.com/blog/2012/09/10/short-read-sequencing/">
  <link href="/favicon.png" rel="icon">
  <link href="/stylesheets/screen.css" media="screen, projection" rel="stylesheet" type="text/css">
  <script src="/javascripts/modernizr-2.0.js"></script>
  <script src="/javascripts/ender.js"></script>
  <script src="/javascripts/octopress.js" type="text/javascript"></script>
  <link href="/atom.xml" rel="alternate" title="Bing & KK" type="application/atom+xml">
  <!--Fonts from Google"s Web font directory at http://google.com/webfonts -->
<link href="http://fonts.googleapis.com/css?family=PT+Serif:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">

<!-- mathjax config similar to math.stackexchange -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
      }
    });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Queue(function() {
        var all = MathJax.Hub.getAllJax(), i;
        for(i=0; i < all.length; i += 1) {
            all[i].SourceElement().parentNode.className += ' has-jax';
        }
    });
</script>

<script type="text/javascript"
   src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>

  

</head>

<body   >
  <header role="banner"><hgroup>
  <h1><a href="/">Bing & KK</a></h1>
  
    <h2>Conding with Zen</h2>
  
</hgroup>

</header>
  <nav role="navigation"><ul class="subscription" data-subscription="rss">
  <li><a href="/atom.xml" rel="subscribe-rss" title="subscribe via RSS">RSS</a></li>
  
</ul>
  
<form action="http://google.com/search" method="get">
  <fieldset role="search">
    <input type="hidden" name="q" value="site:BingW.github.com" />
    <input class="search" type="text" name="q" results="0" placeholder="Search"/>
  </fieldset>
</form>
  
<ul class="main-navigation">
  <li><a href="/">Blog</a></li>
  <li><a href="/blog/archives">Archives</a></li>
</ul>

</nav>
  <div id="main">
    <div id="content">
      <div>
<article class="hentry" role="article">
  
  <header>
    
      <h1 class="entry-title">Short-read Sequencing</h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-10T14:23:00-05:00" pubdate data-updated="true">Sep 10<span>th</span>, 2012</time>
        
      </p>
    
  </header>


<div class="entry-content"><h3 id="sequencing-and-de-novo-assembly">Sequencing and de novo assembly</h3>

<p><a href="http://www.ncbi.nlm.nih.gov/pubmed/21307932">Sequencing technology</a>
<img src="http://bingw.github.com/images/short_read_sequencing_t1.png" alt="Sequencing technology" /></p>

<p>concept: mate pair VS paired end</p>

<h4 id="the-process">The Process:</h4>
<ol>
  <li>Choose sequencing platform</li>
  <li>Make DNA</li>
  <li>Prepare Library</li>
  <li>sequence Run</li>
  <li>primary Analysis Pipeline
 Converts image files (Tb) to sequences (100’s Gb).(inreversbale)</li>
  <li>Obtain Sequence Reads</li>
</ol>

<h4 id="sequence-quality">Sequence quality:</h4>

<p>Phred Score VS accuracy:</p>

<ul>
  <li>10 ==&gt; 90%</li>
  <li>20 ==&gt; 99%</li>
  <li>30 ==&gt; 99.9%</li>
  <li>40 ==&gt; 99.99%</li>
  <li>50 ==&gt; 99.999%</li>
</ul>

<h4 id="format">Format:</h4>

<p>different identifier for phred<a href="http://en.wikipedia.org/wiki/FASTQ_format">wiki</a></p>

<p>A software for this<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/">FastQC</a></p>

<h4 id="genome-scale-sequence-analysis">Genome-scale Sequence Analysis</h4>

<ul>
  <li><strong>De nove assembly</strong></li>
  <li>Templated assembly</li>
  <li>Read mapping or alignment to a reference genome</li>
</ul>

<h4 id="assemblers">Assemblers</h4>

<p>overlap graph assemblers:</p>

<ul>
  <li>Screen out known repeats and set them aside for later</li>
  <li>Infer repetitiveness based on coverage</li>
  <li>First assemble unambiguous overlaps, then resolve repeats using mate pairs</li>
</ul>

<p>De Bruijn K-mers assemblers problems:</p>

<ul>
  <li>Require large amount of memory to store graph</li>
  <li>Not as easy to parallelize as overlap assemblers</li>
</ul>

<h4 id="how-many-clonesreads-do-we-need">How many clones/reads do we need?</h4>

<ul>
  <li>G = Genome Length</li>
  <li>L = Sequence Read Length</li>
  <li>N = Number of Sequences Collected</li>
  <li>T = Number of Basepairs of Overlap Needed</li>
</ul>

<p>Number of Contigs = $N \cdot e^{-\frac{LN}{G} \cdot (1-\frac{T}{L})}$
(<a href="http://www.ncbi.nlm.nih.gov/pubmed/3294162">LW prediction</a>)</p>

<h4 id="gap-closure-strategies">Gap Closure Strategies</h4>

<ul>
  <li>Primer walk to sequence the rest of linking clones that span a scaffold gap</li>
  <li>Primer walk off clones at the ends of contigs for which there is no linking information</li>
  <li>PCR based on your best guess at contig order (comparison to other closely related genomes, predicted genes at the end of genomes, anything else you can come up with)</li>
  <li>Combinatorial PCR with primers designed at the end of each contig</li>
</ul>

</div>


  <footer>
    <p class="meta">
      
  

<span class="byline author vcard">Posted by <span class="fn">BingW</span></span>

      








  


<time datetime="2012-09-10T14:23:00-05:00" pubdate data-updated="true">Sep 10<span>th</span>, 2012</time>
      

<span class="categories">
  
    <a class='category' href='/blog/categories/bioinfo/'>Bioinfo</a>, <a class='category' href='/blog/categories/gen875/'>Gen875</a>
  
</span>


    </p>
    
      <div class="sharing">
  
  
  
</div>

    
    <p class="meta">
      
        <a class="basic-alignment left" href="/blog/2012/09/10/schatz2010-assemblyreview/" title="Previous Post: Schatz2010_AssemblyReview">&laquo; Schatz2010_AssemblyReview</a>
      
      
        <a class="basic-alignment right" href="/blog/2012/09/12/short-read-mapping/" title="Next Post: Short-read mapping">Short-read mapping &raquo;</a>
      
    </p>
  </footer>
</article>

</div>

<aside class="sidebar">
  
    <section>
  <h1>Recent Posts</h1>
  <ul id="recent_posts">
    
      <li class="post">
        <a href="/blog/2012/09/24/structural-functional-annotation/">structural functional annotation</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/20/biochem-note-1/">biochem note 1</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/19/genome-alignment-and-rna-seq/">Genome alignment and RNA-seq</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/18/colorful-python/">让你的python亮起来</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/17/variant-discovery-using-ngs/">Variant discovery using NGS</a>
      </li>
    
  </ul>
</section>

<section>
  <h1>GitHub Repos</h1>
  <ul id="gh_repos">
    <li class="loading">Status updating...</li>
  </ul>
  
  <a href="https://github.com/bingw">@bingw</a> on GitHub
  
  <script type="text/javascript">
    $.domReady(function(){
        if (!window.jXHR){
            var jxhr = document.createElement('script');
            jxhr.type = 'text/javascript';
            jxhr.src = '/javascripts/libs/jXHR.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(jxhr, s);
        }

        github.showRepos({
            user: 'bingw',
            count: 0,
            skip_forks: true,
            target: '#gh_repos'
        });
    });
  </script>
  <script src="/javascripts/github.js" type="text/javascript"> </script>
</section>






  
</aside>


    </div>
  </div>
  <footer role="contentinfo"><p>
  Copyright &copy; 2012 - BingW -
  <span class="credit">Powered by <a href="http://octopress.org">Octopress</a></span>
</p>

</footer>
  











</body>
</html>
