
<!DOCTYPE html>
<!--[if IEMobile 7 ]><html class="no-js iem7"><![endif]-->
<!--[if lt IE 9]><html class="no-js lte-ie8"><![endif]-->
<!--[if (gt IE 8)|(gt IEMobile 7)|!(IEMobile)|!(IE)]><!--><html class="no-js" lang="en"><!--<![endif]-->
<head>
  <meta charset="utf-8">
  <title>structural functional annotation - Bing & KK</title>
  <meta name="author" content="BingW">

  
  <meta name="description" content="Annotaion Structural Annotation - undaries of features of interest (coding regions, regulatory elements, functional RNAs, etc). Ab initio - &hellip;">
  

  <!-- http://t.co/dKP3o1e -->
  <meta name="HandheldFriendly" content="True">
  <meta name="MobileOptimized" content="320">
  <meta name="viewport" content="width=device-width, initial-scale=1">

  
  <link rel="canonical" href="http://BingW.github.com/blog/2012/09/24/structural-functional-annotation/">
  <link href="/favicon.png" rel="icon">
  <link href="/stylesheets/screen.css" media="screen, projection" rel="stylesheet" type="text/css">
  <script src="/javascripts/modernizr-2.0.js"></script>
  <script src="/javascripts/ender.js"></script>
  <script src="/javascripts/octopress.js" type="text/javascript"></script>
  <link href="/atom.xml" rel="alternate" title="Bing & KK" type="application/atom+xml">
  <!--Fonts from Google"s Web font directory at http://google.com/webfonts -->
<link href="http://fonts.googleapis.com/css?family=PT+Serif:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">
<link href="http://fonts.googleapis.com/css?family=PT+Sans:regular,italic,bold,bolditalic" rel="stylesheet" type="text/css">

<!-- mathjax config similar to math.stackexchange -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    tex2jax: {
      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
      processEscapes: true
    }
  });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Config({
      tex2jax: {
        skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
      }
    });
</script>

<script type="text/x-mathjax-config">
    MathJax.Hub.Queue(function() {
        var all = MathJax.Hub.getAllJax(), i;
        for(i=0; i < all.length; i += 1) {
            all[i].SourceElement().parentNode.className += ' has-jax';
        }
    });
</script>

<script type="text/javascript"
   src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
</script>

  

</head>

<body   >
  <header role="banner"><hgroup>
  <h1><a href="/">Bing & KK</a></h1>
  
    <h2>Conding with Zen</h2>
  
</hgroup>

</header>
  <nav role="navigation"><ul class="subscription" data-subscription="rss">
  <li><a href="/atom.xml" rel="subscribe-rss" title="subscribe via RSS">RSS</a></li>
  
</ul>
  
<form action="http://google.com/search" method="get">
  <fieldset role="search">
    <input type="hidden" name="q" value="site:BingW.github.com" />
    <input class="search" type="text" name="q" results="0" placeholder="Search"/>
  </fieldset>
</form>
  
<ul class="main-navigation">
  <li><a href="/">Blog</a></li>
  <li><a href="/blog/archives">Archives</a></li>
</ul>

</nav>
  <div id="main">
    <div id="content">
      <div>
<article class="hentry" role="article">
  
  <header>
    
      <h1 class="entry-title">Structural Functional Annotation</h1>
    
    
      <p class="meta">
        








  


<time datetime="2012-09-24T14:27:00-05:00" pubdate data-updated="true">Sep 24<span>th</span>, 2012</time>
        
      </p>
    
  </header>


<div class="entry-content"><h4 id="annotaion">Annotaion</h4>

<ul>
  <li>Structural Annotation - undaries of features of 
interest (coding regions, regulatory elements, functional 
RNAs, etc). 
    <ul>
      <li>Ab initio - computationally predicted</li>
      <li>Comparative - based on similarity to other genes or genomes</li>
      <li>Experimental - transcript sequencing</li>
    </ul>
  </li>
  <li>Functional Annotation - attaching meaning to the features 
(names, product, activity, biological role, etc.)
    <ul>
      <li>Sequence homology</li>
      <li>Structural similarity or structural features</li>
      <li>Experimental data - gene or protein expression patterns</li>
    </ul>
  </li>
</ul>

<h5 id="errors">Errors</h5>

<ul>
  <li>Structural Annotation Errors
    <ul>
      <li>Wrong coordinates for the gene (exons, regulatory sites, 
etc.)</li>
      <li>False positives - not really a gene (etc.) at all</li>
      <li>False negatives - true genes (etc.) missing</li>
    </ul>
  </li>
  <li>Functional Annotation Errors
    <ul>
      <li>Overly specific function attributed</li>
      <li>Wrong function attributed</li>
      <li>True (known) function missed</li>
      <li>Typos and misspellings</li>
    </ul>
  </li>
</ul>

<h5 id="quality">Quality</h5>

<ul>
  <li>
    <p>Model organisms have content rich structural and 
functional annotations developed and updated over 
time combining <code>computational predictions</code> with 
<code>experimental data</code> and <code>manual curation</code>.</p>
  </li>
  <li>
    <p>Non-model organisms can be well annotated, but 
many are not.</p>
  </li>
  <li>
    <p>Standards for quality and content are still under 
active development and adherence to the standards 
is voluntary.</p>
  </li>
</ul>

<h4 id="annotation-source">annotation source</h4>

<ul>
  <li>From an INSD (International Nucleotide 
Sequence Database) provider
    <ul>
      <li>Genbank</li>
      <li>ENA</li>
      <li>DDBJ</li>
    </ul>
  </li>
  <li>From a “Third Party” provider
    <ul>
      <li>GMOD</li>
      <li>Other specialized databases</li>
    </ul>
  </li>
</ul>

<h4 id="approaches">approaches</h4>

<ul>
  <li>Ab initio
    <ul>
      <li>Based exclusively on computational models</li>
      <li>Error prone, esp. for eukaryotes</li>
      <li>Generally requires manual clean up</li>
    </ul>
  </li>
  <li>Comparative
    <ul>
      <li>Find genes corresponding to sequenced transcripts</li>
      <li>Find the genes already predicted for a closely related 
 organism</li>
    </ul>
  </li>
</ul>

<h4 id="hmms-and-annotation">HMMs and annotation</h4>

<ul>
  <li>Markov models are used in many bioinformatics 
applications</li>
  <li>In particular, Hidden Markov Models are used for
    <ul>
      <li>Ab initio gene prediction</li>
      <li>Ensemble methods for combining results of multiple gene 
 prediction methods</li>
      <li>Protein family similarity searches</li>
      <li>Prediction of other types of features (such as regulatory 
 binding sites)</li>
    </ul>
  </li>
</ul>

<h5 id="using-hmms">Using HMMs</h5>

<ul>
  <li>Given the parameters of the model, compute the probability 
    of a particular output sequence. This problem is solved by the 
    <em>forward algorithm</em>.</li>
  <li>Given the parameters of the model, find the most likely 
    sequence of hidden states that could have generated a given 
    output sequence. This problem is solved by the <em>Viterbi 
    algorithm</em>.</li>
  <li>Given an output sequence or a set of such sequences, find the 
    most likely set of state transition and output probabilities. In 
    other words, train the parameters of the HMM given a 
    dataset of sequences. This problem is solved by the <em>BaumWelch algorithm</em>.</li>
</ul>

<p><a href="http://www.nature.com/nbt/journal/v22/n10/full/nbt1004-1315.html">HMM introduction</a></p>

<h5 id="models-and-training-sets">Models and Training Sets</h5>

<ul>
  <li>The parameters of models are determined from a 
    training set of known positives and negatives, for 
    example a set of known genes, and regions known to 
    not encode genes.</li>
  <li>Size and quality of training sets impacts the 
    performance of models</li>
  <li>Size and quality of training sets is related to the 
    specific genome under investigation</li>
</ul>

<h4 id="the-gene-ontology-go">The Gene Ontology (GO)</h4>

<ul>
  <li>Goal = consistent annotation of gene products within 
    and between organisms</li>
  <li>Gene Ontology Consortium began as a collaboration 
    among model organism dbs (FlyBase, SGD, MGD). 
    Now includes larger number of members and 
    interest groups</li>
  <li>Ontology = A formal representation of concepts and 
    the relationships among them</li>
</ul>

</div>


  <footer>
    <p class="meta">
      
  

<span class="byline author vcard">Posted by <span class="fn">BingW</span></span>

      








  


<time datetime="2012-09-24T14:27:00-05:00" pubdate data-updated="true">Sep 24<span>th</span>, 2012</time>
      

<span class="categories">
  
    <a class='category' href='/blog/categories/bioinfo/'>Bioinfo</a>, <a class='category' href='/blog/categories/gen875/'>Gen875</a>
  
</span>


    </p>
    
      <div class="sharing">
  
  
  
</div>

    
    <p class="meta">
      
        <a class="basic-alignment left" href="/blog/2012/09/20/biochem-note-1/" title="Previous Post: biochem note 1">&laquo; biochem note 1</a>
      
      
    </p>
  </footer>
</article>

</div>

<aside class="sidebar">
  
    <section>
  <h1>Recent Posts</h1>
  <ul id="recent_posts">
    
      <li class="post">
        <a href="/blog/2012/09/24/structural-functional-annotation/">structural functional annotation</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/20/biochem-note-1/">biochem note 1</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/19/genome-alignment-and-rna-seq/">Genome alignment and RNA-seq</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/18/colorful-python/">让你的python亮起来</a>
      </li>
    
      <li class="post">
        <a href="/blog/2012/09/17/variant-discovery-using-ngs/">Variant discovery using NGS</a>
      </li>
    
  </ul>
</section>

<section>
  <h1>GitHub Repos</h1>
  <ul id="gh_repos">
    <li class="loading">Status updating...</li>
  </ul>
  
  <a href="https://github.com/bingw">@bingw</a> on GitHub
  
  <script type="text/javascript">
    $.domReady(function(){
        if (!window.jXHR){
            var jxhr = document.createElement('script');
            jxhr.type = 'text/javascript';
            jxhr.src = '/javascripts/libs/jXHR.js';
            var s = document.getElementsByTagName('script')[0];
            s.parentNode.insertBefore(jxhr, s);
        }

        github.showRepos({
            user: 'bingw',
            count: 0,
            skip_forks: true,
            target: '#gh_repos'
        });
    });
  </script>
  <script src="/javascripts/github.js" type="text/javascript"> </script>
</section>






  
</aside>


    </div>
  </div>
  <footer role="contentinfo"><p>
  Copyright &copy; 2012 - BingW -
  <span class="credit">Powered by <a href="http://octopress.org">Octopress</a></span>
</p>

</footer>
  











</body>
</html>
